lib/CybsNameValuePairClient.php

```diff
@@ -11,9 +11,11 @@
 class CybsNameValuePairClient extends CybsClient
 {
 
-    function __construct($options=array())
+    function __construct($options=array(), $properties=array())
```

Comment:
I would also change the order of the parameters to:

__construct($properties=array(), $options=array())

This way I do not need to pass empty objects to the constructor when I am not using the $options parameter.